src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
+	console.log('Generating DSA....');
+	this.privKey = new DSA();
+	console.log('OK, key generated');
+}
+
+/** Initialize an OTR session for query chans
+ *
+ * Event handler, to be called on chan creation
+ */
+OtrStore.prototype.initChan = function(args) {
+	var network_id = args.network;
+	var chan = args.chan;
+	if (chan.type == Chan.Type.QUERY) {
+		var network = _.find(this.client.networks, {id: network_id});
+		var session = this.getSession(chan.name, network);
+		var irc = network.irc;
+		if (session == undefined) {
+			session = new OTR({
+				fragment_size: 140,
+				send_interval: 200,
```

Comment:
For `fragment_size`, that's the only size I found working (even with [node-otr4 lib](https://github.com/mnaamani/node-otr4)), for `send_interval`, it was from the otr lib README.md but testing a bit more showed me that a lower value can fit and makes the whole thing faster. Thanks for questioning about that :)